devops/kubernetes/storage-node/index.ts

```diff
@@ -217,21 +161,65 @@ const deployment = new k8s.apps.v1.Deployment(
                   name: 'DEBUG',
                   value: 'joystream:*',
                 },
+                {
+                  name: 'COLOSSUS_PORT',
+                  value: `${colossusPort}`,
+                },
+                {
+                  name: 'QUERY_NODE_HOST',
```

Comment:
this has now changed to QUERY_NODE_ENDPOINT and is a full Url not just the host-name